app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientView.kt

```diff
@@ -180,6 +183,14 @@ class LinkIdWithPatientView(
         .build()
   }
 
+  override fun renderPatientName(patientName: String) {
+    idPatientNameTextView.text = Truss()
+        .append(resources.getString(R.string.linkidwithpatient_patient_text))
+        .append(patientName)
+        .append("?")
+        .build()
```

Comment:
We are building this app in languages where the question mark may not be a thing or it may be present at the start of the sentence (in the event of needing to support left-to-right languages).

As such, we should be using string resources to represent this and not hard-code this in source code.